cypress/support/step-definitions/drawer-steps.js

```diff
@@ -0,0 +1,29 @@
+import {
+  drawerToggle,
+  drawerSidebar,
+  drawerSidebarContent,
+} from '../../locators/drawer';
+
+When('I click on Drawer arrow', () => {
+  drawerToggle().first().click();
```

Comment:
try to use `for` loop to pass how many times you want to click this arrow. In feature file you have doubled steps in When/ And

Reply:
fixed 4d965d0